src/sample_submission_data.R

```diff
@@ -0,0 +1,9 @@
+## Script that generates sample submission data to pass validation checks
+require(arrow) # for parquet writing
+source('sim_model_output.R')
+# Function found in /src to generate data frame
+df_sim <- sim_model_output()
+
+# Note: make sure to point to the correct path for your data, defaults to folder 'model-output'
+arrow::write_parquet(df_sim, "auxiliary-data/example-files/2024-10-02-umass-validtestsubmission.parquet")
```

Comment:
In conjunction with https://github.com/reichlab/variant-nowcast-hub/pull/96/files#r1793754309, this will make sure the parquet file is always written to the `auxiliary-data/` folder in the root of the hub.


```suggestion
arrow::write_parquet(df_sim, here::here("auxiliary-data/example-files/2024-10-02-umass-validtestsubmission.parquet"))
```

Reply:
nice--TIL!